docs/API.md

```diff
@@ -108,7 +108,7 @@ Default: `0777`
 
 Octal permission string specifying mode for any folders that need to be created for output folder.
 
-### gulp.task(name[, deps], fn)
+### gulp.task(name [, deps] | [, deps[, cb]] | [, cb])
```

Comment:
I don't think we need the `|` and each signature as separate because the `[ ]` syntax is commonplace for optional